fastplotlib/layouts/_gridplot.py

```diff
@@ -360,8 +365,18 @@ def show(
                 0, 0
             ].camera.maintain_aspect
 
+        # validate vbox if not None
+        if vbox is not None:
+            for widget in vbox:
+                if not isinstance(widget, Widget):
```

Comment:
I don't think we can use `Widget` or any ipywidget imports directly within `show()` because if `ipywidgets` is not installed in the environment (if they are using fastplotlib with only `glfw` or `Qt`), this will raise a error.

Reply:
I would just put it in the `VBox` and let `VBox` deal with it if it's not the right type.